### PR TITLE
ft-158501752] add offer page

### PR DESCRIPTION
**Pull request by Ogollah**
#### What does this PR do?
Create offer page

#### Description of Task to be completed
A user should be able to see requested rides
A user should be able to accept requested ride
A user should be able to reject requested ride

#### How should I test it manually?
Clone this repository
Open your terminal
cd Ride-my-way
git checkout ft-offer-158501752
Open offer.html using your favorite browser

#### What PT stories?
[ft-158501752](https://www.pivotaltracker.com/story/show/158501752)
#### screenshot

offer-new

### CODE REVIEW

**Comment by artorious** [comment]
Looks good.
My input is purely aesthetic, I feel like your color scheme is good but losing to your background image. Consider tweaking/playing with the colors to find a  scheme that works well with a the background(or dull the background). That aside, great work.

**Comment by Ogollah** [comment]
@artorious how about now?

**Comment by meshack-mbuvi** [comment]
It looks awesome

File: UI/templates/offer.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    </div>
+
+    <!--footer fixed-->
+    <div class="footer">
+        <p>Ride-my-way &copy; 2018</p>
+    </div>
+    <script language="JavaScript" src="../static/js/main.js" type="text/javascript"></script>
+</body>
+
+</html>
```
Comment (LWanjiru) [inline UI/templates/offer.html]: Add a new line here

**Review by LWanjiru — CHANGES_REQUESTED** [review]
@Ogollah I agree with @artorious on this, I also recommend making your content fill the page, the empty space on the right side of the page takes away from the great UI I can see on the left.  What is the importance of the blank space on the right? What role does the background play in this page?

**Comment by LWanjiru** [comment]
Please resolve the conflicts @Ogollah. Let me know when you have so I can review it

**Comment by LWanjiru** [comment]
UI looks much better now.

**Pull request merged**